common/djangoapps/third_party_auth/tests/specs/base.py

```diff
@@ -86,23 +85,36 @@ def assert_register_response_in_pipeline_looks_correct(self, response, pipeline_
             if prepopulated_form_data in required_fields:
                 self.assertContains(response, form_field_data[prepopulated_form_data])
 
-    # pylint: disable=invalid-name
-    def assert_account_settings_context_looks_correct(self, context, duplicate=False, linked=None):
-        """Asserts the user's account settings page context is in the expected state.
+    def _get_user_providers_state(self, request):
+        data = {
+            'auth': {},
+        }
+        data['duplicate_provider'] = pipeline.get_duplicate_provider(messages.get_messages(request))
+        auth_states = pipeline.get_provider_user_states(request.user)
+        data['auth']['providers'] = [{
+            'name': state.provider.name,
+            'connected': state.has_account,
+        } for state in auth_states if state.provider.display_for_login or state.has_account]
+        return data
 
-        If duplicate is True, we expect context['duplicate_provider'] to contain
+    # pylint: disable=invalid-name
+    def assert_third_party_accounts_state(self, request, duplicate=False, linked=None):
+        """
+        Asserts the user's third party account in the expected state.
```

Comment:
Docstring title should be separated from docstring body by the new line

Reply:
updated